Sources/Data/OAMapStyleSettings.h

```diff
@@ -66,4 +64,6 @@ typedef NS_ENUM(NSInteger, OAMapStyleValueDataType)
 - (void) save:(OAMapStyleParameter *)parameter;
 - (void) save:(OAMapStyleParameter *)parameter refreshMap:(BOOL)refreshMap;
 
+-(void) resetMapStyleForAppMode:(NSString *)appModeName;
```

Comment:
appModeName is mapPresetName for OAMapStyleSettings. Thus there should not be confusion with names in one class. Looks like method should be: 
`-(void) resetMapStyleForPresetName:(NSString *)mapPresetName;`